src/constants.ts

```diff
@@ -34,28 +34,32 @@ export const SCHEDULED_ORDERS_EXECUTOR_ADDRESS: Address =
   '0x40dc90D670C89F322fa8b9f685770296428DCb6b'
 export const SCHEDULED_TRANSFERS_EXECUTOR_ADDRESS: Address =
   '0xA8E374779aeE60413c974b484d6509c7E4DDb6bA'
+
+// Temp
 export const SMART_SESSIONS_ADDRESS: Address =
-  '0x00000000002B0eCfbD0496EE71e01257dA0E37DE'
+  '0x2358e7436B2bC3F8a82B4F128236a7AF1b32f23c'
+  //'0x0000000000b1c0591b3b3bdadd1e37c8903bd962'
```

Comment:
Could you please update with the latest address: `0x00000000008bDABA73cD9815d79069c247Eb4bDA`

Reply:
nice catch!